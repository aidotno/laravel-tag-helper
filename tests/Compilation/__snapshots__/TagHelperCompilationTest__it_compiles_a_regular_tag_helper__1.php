<?php

return '<div>
    <div>

    Processed</div>
</div>';
