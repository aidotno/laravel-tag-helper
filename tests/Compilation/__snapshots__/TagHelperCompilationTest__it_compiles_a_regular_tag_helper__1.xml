<?xml version="1.0"?>
<div>
  <div>

    Processed</div>
</div>
